app/src/main/java/com/example/poste/models/User.java

```diff
@@ -139,6 +139,22 @@ private void addFolder(Folder newFolder) {
         folders.add(newFolder);
     }
 
+    /**
+     * Adds a folder to the user's folders.
+     *
+     * @param folderId string containing the id of the folder
+     * @return List of posts in the folder
```

Comment:
Perhaps a copy-paste mismatch, this docstring doesn't appear to match the method

Reply:
Yes, fixed it since